man/adoc/bpftrace.adoc

```diff
@@ -3744,6 +3745,12 @@ Default: 1000000
 
 Log size in bytes.
 
+==== log_warnings
+
+Default: 1
+
+Print warnings to the log. If set to 0, this is similar to using the `--no-warnings` CLI option.
```

Comment:
Similar or the same?

Reply:
well... I hope it's the same but because of where they get read/applied we might miss some warnings. I'll just change it to the same.